### ISSUE TITLE
NTR, pulmo: Non-necrotizing pulmonary granulomatosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6384]
**Preferred term label:**
Non-necrotizing pulmonary granulomatosis

**Synonyms**
Non-caseatingpulmonary granulomatosis

**Definition (free text, please give PubMed ID)**
A granuloma that is not associated with necrotic changes

**Parent term (use hpo.jax.org/app)**
Pulmonary granulomatosis HP:0030250

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
DIP ORPHA:98852, HP ORPHA:31740, LIP ORPHA:79128, chronic eosinophilic pneumonia ORPHA:2902, Langerhans cell granulomatosis (Langerhans cell histiocytosis) ORPHA:389

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
added term
